src/app/options/time/time.component.html

```diff
@@ -241,6 +241,19 @@ <h4 class="span-2">{{'options.time.binary' | translate}}</h4>
           <label [for]="'binarydim'+i">{{'options.time.binaryDim' | translate}}<tooltip text="{{'options.time.binaryDimDesc' | translate}}"></tooltip></label>
           <options-toggle type="toggle" [name]="'binarydim'+i" [id]="'binarydim'+i" [(ngModel)]="clock.binary.dim"></options-toggle>
         </div>
+        <div class="input" *ngIf="clock.binary.enabled === true">
+          <label [for]="'binaryUseNumbers'+i">{{'options.time.binaryUseNumbers' | translate}}<tooltip text="{{'options.time.binaryUseNumbers' | translate}}"></tooltip></label>
+          <options-toggle type="toggle" [name]="'binaryUseNumbers'+i" [id]="'binaryUseNumbers'+i" [(ngModel)]="clock.binary.useNumbers"></options-toggle>
+        </div>
+        <div class="input" *ngIf="clock.binary.useNumbers === true">
```

Comment:
You should be checking if useNumbers is true as well as binary is enabled, otherwise you could have the on/off values show when binary is disabled. Same for the off value.